index.js

```diff
@@ -40,7 +40,8 @@ const plugins = {
   spawn_point: require('./lib/plugins/spawn_point'),
   tablist: require('./lib/plugins/tablist'),
   time: require('./lib/plugins/time'),
-  villager: require('./lib/plugins/villager')
+  villager: require('./lib/plugins/villager'),
+  util: require('./lib/plugins/util')
```

Comment:
util is a name that should be avoided
this could go in physics or time files

Reply:
Okay. What should I call it? Helper?

Reply:
helper has the same issue as util: it can contain anything and everything
why not put this function in physic.js as this is where physicTick is defined ?

Reply:
Ooh, I misunderstood you. Okay, thank makes sense. At first I thought you said there was a file called util that contained physics based utility functions.

I'll move it to physics, sure.

Reply:
Done.